contract/tools/ui-kit-goals/marshalTables.js

```diff
@@ -9,12 +9,6 @@
 // @ts-check
 import { Far, makeMarshal } from '@endo/marshal';
 
-/**
- * The null slot indicates that identity is not intended to be preserved.
- *
- * @typedef { string | null } WildSlot
```

Comment:
oic, `WildSlot` predates this PR.

I think it's helpful to have this type named so we can keep the explanation about what `null` indicates.

To find a better name I went searching in agoric-sdk and it turns out that also has `WildSlot`.
https://github.com/Agoric/agoric-sdk/blob/2d48e143f54911897e8345e1094ae6d19599bef2/multichain-testing/tools/marshalTables.js#L15

@dckc what's the idea behind the name? WDYT of something like `OptionalSlot` or `MaybeSlot` or `BoardSlot`?

Reply:
"is not intended to be _preserved_" maybe `TempSlot` as in, its temporary. I'm fine with whatever you agree on and decide.

Reply:
I suppose `WildSlot` meant "either a slot or a wildcard". I like `MaybeSlot`. I'm a fan of haskell and the [Typeclassopedia](https://wiki.haskell.org/Typeclassopedia). But when in Rome... I just checked [some typescript docs](https://www.typescriptlang.org/docs/handbook/utility-types.html); there I see "optional". So `OptionalSlot`, please.